src/components/__snapshots__/ItemList.test.js.snap

```diff
@@ -0,0 +1,75 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`should display a list of items 1`] = `
```

Comment:
Nice snapshots! I imagine these will need to be rerun as designs come in.

Reply:
Thx!! Yup, they will.